dist/rpm/openQA.spec

```diff
@@ -553,12 +553,14 @@ fi
 %dir %{_sysconfdir}/apache2
 %dir %{_sysconfdir}/apache2/vhosts.d
 %config %{_sysconfdir}/apache2/vhosts.d/openqa.conf.template
-%config %{_sysconfdir}/apache2/vhosts.d/openqa-common.inc
+%config(noreplace) %{_sysconfdir}/apache2/vhosts.d/openqa-common.inc
 %config %{_sysconfdir}/apache2/vhosts.d/openqa-ssl.conf.template
 # nginx vhost
 %dir %{_sysconfdir}/nginx
 %dir %{_sysconfdir}/nginx/vhosts.d
-%config %{_sysconfdir}/nginx/vhosts.d/openqa.conf
+%config %{_sysconfdir}/nginx/vhosts.d/openqa.conf.template
+%config(noreplace) %{_sysconfdir}/nginx/vhosts.d/opeenqa-locations.inc
+%config(noreplace) %{_sysconfdir}/nginx/vhosts.d/opeenqa-upstreams.inc
```

Comment:
```suggestion
%config(noreplace) %{_sysconfdir}/nginx/vhosts.d/openqa-locations.inc
%config(noreplace) %{_sysconfdir}/nginx/vhosts.d/openqa-upstreams.inc
```

Reply:
and please also describe how you tested your changes, e.g. local osc build and run from there in container as suggested in the ticket, etc.

Reply:
For now I manually copied the files into the config directory of my local nginx server.
But I think we should add some kind of automated test.

Reply:
And please also test against the nginx setup on o3.